contrib/windows/build-installer.iss

```diff
@@ -14,6 +10,9 @@
 #define AppMainExeName "bin\julia.exe"
 #define CurrentYear GetDateTimeString('yyyy', '', '')
 
+#ifndef AppId
+#define AppId AppNameLong
```

Comment:
```suggestion
#define AppId DirName
OR
#define AppId AppName + "-" + AppVersion

```

to avoid the space in the reg key? I think that might make it easier to parse if in the future we add an auto-uninstaller.